src/Native/Unix/System.Native/pal_networking.c

```diff
@@ -2247,7 +2251,32 @@ static int32_t WaitForSocketEventsInner(int32_t port, struct SocketEvent* buffer
 }
 
 #else
-#error Asynchronous sockets require epoll or kqueue support.
+// TODO: Poll fallback
```

Comment:
Is this something you plan to follow-up on?  If no, please remove the TODO.  If yes, please open an issue for it and include the issue number here.

Reply:
I plan to, but since it involves quite a bit of work and I don't think anything on the Mono side uses this yet, not in this PR.

I'll remove this and make the defined stuff more consistent.